hudi-gcp/src/main/java/org/apache/hudi/gcp/bigquery/BigQuerySyncConfig.java

```diff
@@ -58,6 +58,13 @@ public class BigQuerySyncConfig extends HoodieSyncConfig implements Serializable
       .markAdvanced()
       .withDocumentation("Name of the target project in BigQuery");
 
+  public static final ConfigProperty<String> BIGQUERY_SYNC_BILLING_PROJECT_ID = ConfigProperty
+      .key("hoodie.gcp.bigquery.sync.billing.project.id")
+      .noDefaultValue()
+      .sinceVersion("1.0.0")
+      .markAdvanced()
+      .withDocumentation("Name of the billing project id in BigQuery. By default it will use the BIGQUERY_SYNC_PROJECT_ID. This can only be used with manifest file based approach");
```

Comment:
```suggestion
      .withDocumentation("Name of the billing project id in BigQuery. By default it uses the configuration from `BIGQUERY_SYNC_PROJECT_ID` (`hoodie.gcp.bigquery.sync.project_id`) if this configuration is not set. This can only be used with manifest file based approach");
```